ExeterTeam25/group-software-project
# Comments2

There is now a page to view all comments for a response:

To test:

Add some comments to a response and check they now show with most recent at the top
Then click on see all responses and make sure it all shows properly
